teamcity/tests/conftest.py

```diff
@@ -2,13 +2,46 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import pytest
+from six import PY3
 
 from datadog_checks.dev import docker_run
+from datadog_checks.teamcity import TeamCityCheck
 
-from .common import COMPOSE_FILE, CONFIG
+if PY3:
+    from datadog_checks.teamcity.check import TeamCityCheckV2
+
+from .common import COMPOSE_FILE, CONFIG, USE_OPENMETRICS
 
 
 @pytest.fixture(scope='session')
-def dd_environment():
+def dd_environment(instance, omv2_instance):
     with docker_run(COMPOSE_FILE, sleep=10):
-        yield CONFIG['instances'][0]
+        if USE_OPENMETRICS:
+            yield omv2_instance
+        else:
+            yield instance
+
+
+@pytest.fixture(scope='session')
+def legacy_instance():
+    return CONFIG['instances'][0]
```

Comment:
Small nit, but can you separate out `CONFIG` into 3 separate dictionaries instead?